src/status_im/ui/screens/signing/views.cljs

```diff
@@ -46,16 +46,16 @@
 (defn separator []
   [react/view {:height 1 :background-color colors/gray-lighter}])
 
-(defn acc-text [txt1 txt2]
-  [react/nested-text nil
-   txt1 " "
-   [{:style {:color colors/gray}} txt2]])
-
 (defn displayed-name [contact]
   (if (or (:preferred-name contact) (:name contact))
     (multiaccounts/displayed-name contact)
     (:address contact)))
 
+(defn acc-text [txt1 txt2]
```

Comment:
no point moving this

Reply:
Sorry, human error here.  Was actually just going to suggest removing this since I was replacing the acc-text component with a more generic nested-text element in the sheets function.  From a style perspective, is it preferred to extract these kind of subcomponents out into their own function definition?  If so, I can just repurpose the existing acc-text function to incorporate the additional elements for the currency conversion.

Reply:
I've gone ahead and removed the acc-text function definition since I'm not using it in my PR (which I think is now basically complete) and it's not used anywhere else..  Let me know if you want me to revise things further on this one.